DigitalExcellence/dex-backend
# Feature/229 ability to like projects

## Description

- Added a new endpoint to the API for ability to like a project
- Fixed some of the typo issues
- Trimmed additional spaces.

## Type of change


## Checklist


## Steps to Test or Reproduce

1. Start Postman
2. Create a new request.
3. Select the request type PUT
4. Use "/project/{{projectId}}/like/{{userId}}" as a request template.
5. Run the test
6. You should see the userIds who liked the project in project table.

## Link to issue

Closes: [#229](https://github.com/DigitalExcellence/dex-backend/issues/229)
